dq-server/src/routes/themes.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const Joi = require('joi');
+const jwt = require('jsonwebtoken');
+const config = require('config');
+const themeController = require('../controllers/themes');
+const { createThemeSchema } = require('../validation/input');
+const router = express.Router();
+router.use(express.json());
+
+
+router.get('/', async (req, res) => {
```

Comment:
2 times GET ?

Reply:
One without query params and other with them. Better change it?